### DISCUSSION TITLE
Trending - not trending, but the FYP?

### DISCUSSION THREAD

**Discussion by honzarox** [davidteather/TikTok-Api#1139] [General]
Hi,

wanted to ask about the Trending API. It uses tiktok.com/api/recommend/item_list/, which to my knowledge shows the recommended items from a users timeline.

Do I understand this correctly? Or does it actually fetch the "trending" videos for a certain period of time?

**Comment by Kkordik** [comment]
I guess yes, these videos are actually recommended (or personalized).

As `videos()` method makes such request:

```
            resp = await Trending.parent.make_request(
                url="https://www.tiktok.com/api/recommend/item_list/",
                params=params,
                headers=kwargs.get("headers"),
                session_index=kwargs.get("session_index"),
            )
```

As we can see, these 'trending' videos are actually 'recommended' ones. But as TikTokApi has no login option, these recommendations are for not logined user. It may be similar but not exactly 'trending' videos
